rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -613,6 +613,61 @@
             (BridgeMethodExecutorTyped) Bridge::registerFastBridgeBtcTransaction,
             activations -> activations.isActive(RSKIP176),
             false
+    ),
+    GET_BEST_BLOCK_HEADER(
```

Comment:
```suggestion
    GET_BTC_BLOCKCHAIN_BEST_BLOCK_HEADER(
```

These methods should also be renamed like in Bridge class, to the more explicit form

Reply:
OK, done. But take into account there is a collision of name in `getBtcBlockchainBestChainHeight` there is a previous method with different external behavior (only local calls allowed). Now, I named the new method as `getBtcBlockchainBestChainHeight2` but what is your recommendation?